### ISSUE TITLE
Need a jar-to-war profile instead of amp-to-war

### ISSUE THREAD

**Issue by shazada** [Alfresco/alfresco-sdk#441]
We don't use AMPS to develop and where Alfresco is heading they are also heading towards JARS 'again'.
So it's we need a jar-to-war profile with overlag instead of amp-to-war.

Now if we want to make a deployable jar for Alfresco/Share we add the assembly xml to create an installable jar

```
<plugin>
						<artifactId>maven-assembly-plugin</artifactId>
						<configuration>
							<appendAssemblyId>false</appendAssemblyId>
							<attach>false</attach>
						</configuration>
						<executions>
							<execution>
								<id>make-src-jar</id>
								<phase>package</phase>
								<goals>
									<goal>single</goal>
								</goals>
								<configuration>
									<classifier>src</classifier>
									<finalName>${project.build.finalName}-src</finalName>
									<descriptor>src/main/assembly/jar.xml</descriptor>
								</configuration>
							</execution>
						</executions>
					</plugin>
					<plugin>
						<groupId>org.codehaus.mojo</groupId>
						<artifactId>build-helper-maven-plugin</artifactId>
						<version>1.8</version>
						<executions>
							<execution>
								<id>attach-artifacts</id>
								<phase>package</phase>
								<goals>
									<goal>attach-artifact</goal>
								</goals>
								<configuration>
									<artifacts>
										<artifact>
											<file>${project.build.directory}/${project.build.finalName}-src.jar</file>
											<type>jar</type>
											<classifier>src</classifier>
										</artifact>
									</artifacts>
								</configuration>
							</execution>
						</executions>
					</plugin>
```

Assembly jar:

```
<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.0 http://maven.apache.org/xsd/assembly-1.1.0.xsd">
	<id>installable</id>
	<formats>
		<format>jar</format>
	</formats>
	<includeBaseDirectory>false</includeBaseDirectory>
	<fileSets>
		<fileSet>
			<directory>${project.build.directory}/classes</directory>
			<outputDirectory></outputDirectory>
		</fileSet>
		<fileSet>
			<directory>${project.build.directory}/amp/config/alfresco</directory>
			<outputDirectory>alfresco</outputDirectory>
			<excludes>
				<exclude>**/*.ftl</exclude>
			</excludes>
		</fileSet>
		<fileSet>
			<directory>${project.build.directory}/amp/config/alfresco</directory>
			<outputDirectory>alfresco</outputDirectory>
			<includes>
				<include>**/*.ftl</include>
			</includes>
			<filtered>false</filtered>
		</fileSet>
	</fileSets>
</assembly>
```

So for clean package this works perfectly, now when we want to run our jar code and use the amp-to-war.
We get the following: 
[ERROR] Failed to execute goal org.alfresco.maven.plugin:alfresco-maven-plugin:2.2.0:install (amps-to-war-overlay) on project alfresco: No AMP file(s) found in C:\Projecten\github\green\contezza\alfresco\target\alfresco.amp - AMP installation cannot proceed -> [Help 1]

So we don't build an amp.....and there is only a jar which is already in the overlay of the alfresco.war.

Bottom line :) Make the SDK for jar modules, so it can build & run in the tomcat like the amp-to-war does.
Thanks!

**Comment by ohej** [comment]
SDK 3.0 is just around the corner where JARs are the cornerstone of your project. We've flipped the table: JARs are the default, AMPs are available as an assembly. 

Are you OK if we close this issue as it's all addressed with the upcoming release?

**Comment by shazada** [comment]
Sure! great didn't knew you were already that far 👍
